CONTRIBUTING.md

```diff
@@ -1,4 +1,28 @@
 Contributing
 ============
 
-ADD ACTUAL CONTRIBUTING GUIDLINES
+Thanks for helping out!
```

Comment:
That sounds like a good compromise to me. In that case we should thing of a generic name like `destination-contributing.md` or something. I'll do the generator end.